emcc.py

```diff
@@ -2315,6 +2315,7 @@ def phase_linker_setup(options, state, newargs):
     settings.SYSCALLS_REQUIRE_FILESYSTEM = 0
     settings.JS_LIBRARIES.append((0, 'library_wasmfs.js'))
     settings.REQUIRED_EXPORTS += ['_wasmfs_read_file']
+    settings.REQUIRED_EXPORTS += ['_wasmfs_getcwd']
```

Comment:
This can just be added to the list on the previous line instead of getting its own line.